src/pages/ReportDescriptionPage.js

```diff
@@ -80,39 +80,39 @@ function ReportWelcomeMessagePage(props) {
         <ScreenWrapper
             shouldEnableMaxHeight
             includeSafeAreaPaddingBottom={false}
-            testID={ReportWelcomeMessagePage.displayName}
+            testID={ReportDescriptionPage.displayName}
         >
-            <FullPageNotFoundView shouldShow={ReportUtils.shouldDisableWelcomeMessage(props.report, props.policy)}>
+            <FullPageNotFoundView shouldShow={ReportUtils.shouldDisableReportDescription(props.report, props.policy)}>
                 <HeaderWithBackButton
-                    title={props.translate('welcomeMessagePage.welcomeMessage')}
+                    title={props.translate('reportDescriptionPage.roomDescription')}
                     onBackButtonPress={() => Navigation.goBack(ROUTES.REPORT_SETTINGS.getRoute(props.report.reportID))}
                 />
                 <FormProvider
                     style={[styles.flexGrow1, styles.ph5]}
-                    formID={ONYXKEYS.FORMS.WELCOME_MESSAGE_FORM}
+                    formID={ONYXKEYS.FORMS.REPORT_DESCRIPTION}
```

Comment:
Bug: 
1. console error on opening room description page.
2. setting a room description and navigating to room description crashes the app

https://github.com/Expensify/App/assets/29673073/24731d74-fd11-43e5-99a6-0109b8193add

Reply:
@rushatgabhane As for the first bug, I just noticed it was because of a typo in the `_FORM` at the end. But I wasn't able to reproduce the crash. Reading the stack trace from the crash, it doesn't seem to be related to the code edited... Could you please confirm if it is reproducible reliably?

https://github.com/Expensify/App/assets/77237602/3e3e8a91-e910-4d27-88af-03b51caa1e05